torch/nested/_internal/ops.py

```diff
@@ -1261,6 +1261,41 @@ def _nested_get_max_seqlen(func, *args, **kwargs):
     return inp._metadata_cache.get("max_seqlen", None)
 
 
+@register_jagged_func(torch.ops.aten.masked_select.default, "self: jt, mask: any")
+def masked_select_default(func, *args, **kwargs):
```

Comment:
It's also probably worth documenting the semantics here in a comment (until we find a better place to put it):
- whats our support for inputs beyond 2-D?
- what happens when we fully mask out a row